src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
+  const arrayOfStates = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(newState, action.extraData);
+
+        break;
+      case 'removeProperties':
```

Comment:
```suggestion
        Object.assign(newState, action.extraData);
        break;
        
      case 'removeProperties':
```

Reply:
Is it bad form to put a space before a break? I think it makes the code more readable.

Reply:
Usually we have spaces between cases, break is a part of the previous case, looks a bit weird to have it grouped with the next case.

Reply:
In Tetiana's suggestion, the space is moved to be below `break`.